backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -0,0 +1,34 @@
+package ca.bc.gov.hlth.mohums.controller;
+
+import ca.bc.gov.hlth.mohums.webclient.WebClientService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+import java.util.Optional;
+
+@RestController
+public class UsersController {
```

Comment:
We named the other controllers singular (e.g. GroupController, ClientController). The naming convention should be the same, not sure which way it should go though...

Reply:
renamed to GroupsController and ClientsController